### ISSUE TITLE
increased/decreasesd aortic weight

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3324]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased aortic weight
2. decreased aortic weight

>2. Synonyms (e.g., absent spleen)

2. reduced aortic weight

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. increase in the average weight of the aorta
2. decrease in the average weight of the aorta

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:17143179

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal aortic weight (MP:0002845)
2. abnormal aortic weight (MP:0002845)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683
--

**Comment by anna-anagnostop** [comment]
Added new children terms:
decreased aortic weight MP:0031199
increased aortic weight MP:0031200
